docs/device_api.md

```diff
@@ -14,9 +14,69 @@ working as a client SDK for any desktop PC, laptop or gateway supporting a pytho
 the steps described below and use the python commands as described in the README.md file available at this [Github
 repository](https://github.com/telefonicaid/fiware-figway).
 
+# Configure the South Bound protocol
+
+In order for the South Bound protocols to work for your services, you must provision the information about your
```

Comment:
Is the "South Bound" concept defines somewhere? Can we assume that is widely known and such definition is not required?